draft-tigress-requirements.md

```diff
@@ -105,8 +105,19 @@ This document describes the use cases necessitating the secure transfer of digit
 
 # Introduction
 
-TODO Introduction
+Today, there is no standard way of transferring digital credentials securely between two devices belonging to the same platform or two different platforms. This document describes the problem space and the requirements for the solution the working group creates.
 
+# General Setting
+
+When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two devices and potentially an interemediary server, there are a couple more actors involved. First of all, each device communicates with its own servers. These are referenced as the OEM server or the Device OEM server. In addition, there are the companies that are providing the digital credential. These are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. 
+
+Without Tigress, devices are already connected with their Device OEM Servers and those servers already connect with provisioning partners. We assume that all devices follow this paradigm. 
+
+Device <-> Device OEM <-> Provisioning Partner
```

Comment:
```suggestion
When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two devices, potentially through an interemediary server, there are a couple more actors involved. 

The companies that are providing the digital credential are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. Each device has a preexisting trust relationship between itself and the Provisioning Partner.
```
I don't think we should mention the "Device OEM" unless we  need to talk about something that only the device OEM can do. What do you think of something like this?

I think we can omit the device OEM to provisioning partner interaction, and write it off as an implementation detail.

Reply:
I added your suggestion but my last paragraph still has OEM notes. I think it is helpful, but I can remove it if you don't find it necessary!